docs/install-config/configure-internal-tls.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Configure Internal TLS communication between Harbor Component
+weight: 30
+---
+
+By default, The communication between Harbor's component are use http protocol which regards as insecure in production environment. From Harbor v2.0, TLS can be enabled for internal network. In production environments, always use HTTPS is a best practice.
+
+It is introduced a new item `internal_tls` in `harbor.yml` file. To enabled internal TLS is just set the `enabled` to `true` and set the `dir` value to the path of directory that place internal cert file.
```

Comment:
This functionality is introduced via the 'internal_tls' in 'harbor.yml' file. To enable internal TLS, set `enabled` to `true` and set the `dir` value to the path of directory that contains the internal cert file.